api/envoy/admin/v2alpha/config_dump.proto

```diff
@@ -178,3 +179,43 @@ message RoutesConfigDump {
   // The dynamically loaded route configs.
   repeated DynamicRouteConfig dynamic_route_configs = 3 [(gogoproto.nullable) = false];
 }
+
+// Envoy's scoped RDS implementation fills this message with all currently loaded route
```

Comment:
Thanks @AndresGuedez for this, my main thought looking at this is that the PR is too big to reasonably review and without diluting review bandwidth. Can we split the PR up further for review? 

I think we should have this master PR kept open to show how things tie together, but then separate PRs for:

1. Protos.
2. Scoped RDS configuration parser and handling logic. This can just be unit tested and doesn't need to be wired up yet.
3. Config subscription and wiring of scoped RDS into the rest of Envoy. The integration test can be added here.
4. Optionally, other PRs to reduce the amount of renaming noise, where reasonable.

I think we can get better review velocity if we do it this way.

Reply:
Ack. Working on splitting this up.